src/components/Paginate.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from "react";
 
-const findNumberOfPages = (
+const getPageCount = (
```

Comment:
We renamed the function that finds the number of pages of search results from `findNumberOfPages` to `getPageCount` because `findNumberOfPages` implies use of the .find() method [as amilajack mentioned](https://github.com/amilajack/tide/pull/290#discussion_r546048793).